src/include/catalog/catalog_accessor.h

```diff
@@ -265,6 +265,7 @@ class CatalogAccessor {
   transaction::TransactionContext *txn_;
```

Comment:
Propose const-qualifying catalog_, dbc_, txn_, and db_oid_.

Reply:
Changing `catalog_` and `dbc_` required adding const qualifiers to more functions but was doable.  However, we cannot const qualify `txn_` because it is modified by transactions that write into the catalog (both through adding actions to the transaction's queue and creating redo records).

Reply:
Oh, I meant making the pointer const. transaction::TransactionContext *const txn_;